CONTRIBUTING.md

```diff
@@ -0,0 +1,197 @@
+    cd docs
+    git checkout latest
+    ```
+
+1.  List the current remote branches:
+
+    ```bash
+    git remote -v
+    ```
+
+    This command should list two remotes, both marked `origin`, like this:
+
+    ```bash
+    origin  https://github.com/<username>/docs.git (fetch)
+    origin  https://github.com/<username>/docs.git (push)
+    ```
+
+    The `origin` remotes are your own fork, and you can do whatever you want here without changing the upstream repository.
+
+1.  Add the docs repo as an upstream:
+
+    ```bash
+    git remote add upstream https://github.com/timescale/docs.git
+    ```
+
+1.  Check:
+
+    ```bash
+    git remote -v
+    ```
+
+    This command should now have the same two `origin` remotes as before, plus two more labelled `upstream`, like this:
+
+    ```bash
+    origin  https://github.com/<username>/docs.git (fetch)
+    origin  https://github.com/<username>/docs.git (push)
+    upstream  https://github.com/timescale/docs.git (fetch)
+    upstream  https://github.com/timescale/docs.git (push)
+    ```
+
+1.  Check out your fork's `latest` branch:
+
+    ```bash
+    git checkout latest
+    ```
+
+1.  Fetch the branches in the upstream repository:
+
+    ```bash
+    git fetch upstream
+    ```
+
+1.  Merge the changes from the upstream `latest` branch, into your fork's
+    `latest` branch:
+
+    ```bash
+    git merge upstream/latest
+    ```
+
+1.  Create a new branch for the work you want to do. Make sure you give it an
+    appropriate name, and include your username:
+
+    ```bash
+    git checkout -b update-readme-username
+    ```
+
+
+### Procedure: Committing changes and creating a pull request
+1.  Make your changes.
+1.  Add the updated files to your commit:
+
+    ```bash
+    git add .
+    ```
+
+1.  Commit your changes:
+
+    ```bash
+    git commit -m "Commit message here"
+    ```
+
+1.  Push your changes:
+
+    ```bash
+    git push
+    ```
+
+    If git prompts you to set an upstream in order to push, use this command:
+
+    ```bash
+    git push --set-upstream origin <branchname>
+    ```
+
+1.  Create a pull request (PR) by navigating to <https://github.com/timescale/docs>
+    and clicking `Compare and Create Pull Request`. Write an informative commit
+    message detailing your changes, choose reviewers, and save your PR. If you
+    haven't yet finished the work you want to do, make sure you create a draft PR by
+    selecting it from the drop down box in the GitHub web UI. This lets your
+    reviewers know that you haven't finished work yet, while still being transparent
+    about what you are working on, and making sure we all understand current
+    progress.
+
+
+<highlight type="tip">
+Choose your reviewers carefully! If you have made changes to the technical
+detail of the documentation, choose an appropriate subject matter expert (SME)
+to review those changes. Additionally, every change requires at least one
+documentation team member to approve.
+</highlight>
```

Comment:
This will only be seen/shown on GitHub, correct? If so, I don't think this will highlight anything as it's our own component.

Reply:
oh, good point.